pkg_building.es.Rmd

```diff
@@ -1,513 +1,511 @@
 
-* Find other interesting aspects of naming your package [in this blog post by Nick Tierney](https://www.njtierney.com/post/2018/06/20/naming-things/), and in case you change your mind, find out [how to rename your package in this other blog post of Nick's](https://www.njtierney.com/post/2017/10/27/change-pkg-name/).
+- Encuentra otros aspectos interesantes sobre cómo nombrar tu paquete [en esta entrada del blog de Nick Tierney](https://www.njtierney.com/post/2018/06/20/naming-things/)y, en caso de que cambies de opinión, descubre [cómo cambiar el nombre de tu paquete en esta otra entrada del blog de Nick](https://www.njtierney.com/post/2017/10/27/change-pkg-name/).
 
-### Creating metadata for your package
+### Crear metadatos para tu paquete
 
-We recommend you to use the [`codemetar` package](https://github.com/ropensci/codemetar) for creating and updating a JSON [CodeMeta](https://codemeta.github.io/) metadata file for your package via `codemetar::write_codemeta()`. It will automatically include all useful information, including [GitHub topics](#grooming). CodeMeta uses [Schema.org terms](https://schema.org/) so as it gains popularity the JSON metadata of your package might be used by third-party services, maybe even search engines. 
+Te recomendamos que utilices la función [`codemetar` paquete](https://github.com/ropensci/codemetar) para crear y actualizar un JSON [CodeMeta](https://codemeta.github.io/) para tu paquete a través de `codemetar::write_codemeta()`. Incluirá automáticamente toda la información útil, incluyendo [Temas de GitHub](#grooming). CodeMeta utiliza [términos de Schema.org](https://schema.org/) por lo que, a medida que gane popularidad, los metadatos JSON de tu paquete podrían ser utilizados por servicios de terceros, incluso por motores de búsqueda.
 
+## Plataformas
 
-## Platforms
+- Los paquetes deben funcionar en todas las plataformas principales (Windows, macOS, Linux).  Se pueden conceder excepciones a los paquetes que interactúan con funciones específicas del sistema, o envoltorios para utilidades que sólo funcionan en plataformas limitadas, pero los autores deben hacer todo lo posible para la compatibilidad entre plataformas, incluyendo la compilación específica del sistema, o la contenerización de utilidades externas.
 
-* Packages should run on all major platforms (Windows, macOS, Linux).  Exceptions may be granted packages that interact with system-specific functions, or wrappers for utilities that only operate on limited platforms, but authors should make every effort for cross-platform compatibility, including system-specific compilation, or containerization of external utilities.
```

Comment:
We're having trouble translating "or containerization of external utilities." What does it mean?